etc/schema/mysql.schema.sql

```diff
@@ -86,6 +87,8 @@ CREATE TABLE x509_target (
   hostname varchar(255) NULL DEFAULT NULL,
   latest_certificate_chain_id int(10) unsigned NULL DEFAULT NULL,
   ctime timestamp NULL DEFAULT NULL,
+  last_scan bigint(20) NOT NULL DEFAULT (EXTRACT(EPOCH FROM NOW())),
```

Comment:
This won't work with mysql! Have you tried to import the schema with this new column? :(

Reply:
I think it's perfectly fine to only have `.. NOT NULL`